src/components/application_manager/include/application_manager/service.h

```diff
@@ -218,7 +226,8 @@ class Service {
    * @param message message for validation
    * @return true if message is valid according to schema, otherwise false
    */
-  virtual bool ValidateMessageBySchema(const Message& message) = 0;
+  virtual MessageValidationResult ValidateMessageBySchema(
+      const Message& message) = 0;
```

Comment:
@AByzhynar why do you need `MessageValidationResult` return result for this function? As I could see you are using comparison with `SUCCESS` only so bool would be enough here